FeatureSuggestion/ShaktiPrasad_SystemEntities_Suggestions.md

```diff
@@ -0,0 +1,44 @@
+#### This may not cover all required entities. Just putting forward my thoughts.
+
+#### Class Section
+{
+````
+	string title;
+	string paragraph;(rich text)
+	Public:
+		Section(title, paragraph)
```

Comment:
It's the right design but in order to make the customer experience better, we can make some sections as special sections. Like the Education Section should mention the school name. Degree and when started and when ended.

Reply:
Use inheritance to implement other sections.